phototherapy/index.html

```diff
@@ -51,6 +51,14 @@
   </div>
 </nav>
 <div id="phototherapy-main" class="container main-content phototherapy-main-content">  
+  <div id="usage-terms-dialog-backdrop" class="usage-terms-dialog-backdrop" ng-show="!ctrl.termsSigned">
+    <div id="usage-terms-dialog" class="usage-terms-dialog">
+      <p>
+        בשימושך באתר אתה מצהיר כי הנך חבר צוות רפואי באסותא אשדוד, והנך מבין כי האתר מיועד לשימוש תומך בלבד ואינו מהווה תחליף לשיקול דעת רפואי מקצועי. לחץ על "אני מסכים" על מנת להמשיך.
+      </p>
+      <button type="button" class="btn btn-primary" ng-click="ctrl.acceptTerms()">אני מסכים</button>
```

Comment:
can we align the button to the center?
also, can we make change the colors to either of the followings (i prefer the left one)?

![image](https://github.com/yonatankatz/mashlom-assuta-doctors/assets/23584289/41d50cf4-1af3-42a9-aa4d-289ef219bf3e)